app/views/collections/index.html.erb

```diff
@@ -12,6 +12,7 @@
   <div class="grid gap-2 m:gap-4 l:gap-6 m:grid-cols-2 l:grid-cols-3 px-2 m:px-0">
     <% if @collections.present? %>
       <% @collections.each do |collection| %>
+        <% next if collection.articles.published.empty? %>
```

Comment:
**thought (non-blocking):** I'm wondering if these could already be filtered out in the controller, i.e. should `@collections` only include collections with at least one published article?